### ISSUE TITLE
LARGE_BANNER request

### ISSUE THREAD

**Issue by oangelo** [sunnycupertino/cordova-plugin-admob-simple#10]
Could you implement LARGE_BANNER? Thanks

**Comment by sunnycupertino** [comment]
Thank you for the good suggestion, we have implemented it now. You can activate it by setting LARGE_BANNER in the javascript.

**Comment by oangelo** [comment]
I tested, and I can't get it to work just by changing AdSize in the javascript.
I changed  adSize: window.plugins.AdMob.AD_SIZE.SMART_BANNER to 
adSize: window.plugins.AdMob.AD_SIZE.LARGE_BANNER, but I got a Smart Banner anyway. 
However, it worked when I changed "private AdSize adSize = AdSize.SMART_BANNER;" to "private AdSize adSize = AdSize.LARGE_BANNER;" in /src/android/AdMob.java. But I don't understand why...
